samples/caf_sensor_manager/src/modules/workload_sim.c

```diff
@@ -88,15 +88,12 @@ static bool event_handler(const struct app_event_header *aeh)
 		return false;
 	}
 
-	if (is_module_state_event(aeh)) {
-		const struct module_state_event *event = cast_module_state_event(aeh);
+	if (is_config_event(aeh)) {
+		
+		const struct config_event *event = cast_config_event(aeh);
```

Comment:
please use the established method of communicating that module is ready - creating a new event is not needed here

Reply:
In beginning I tried using module_state_event. The problem is module ids are not transferred between cores. I couldn't than nicely recognize that information that sample should start on second core is from first core. I can send module_state_event independently on both cores but I thought that it will be more clear if one core initialize whole sample.